app.rb

```diff
@@ -135,6 +107,73 @@
   redirect to("/merchant/#{merchant.public_id}")
 end
 
+def _user_and_business(country_code)
```

Comment:
Hmm..this block now takes about about a third of the file (ie: the app).  Could we somehow condense this? Either by Zach's solution, breaking the prefill data into a separate file, or just showing a few fields here?  Otherwise I think we sacrifice readability.